content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +202,62 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before
+continue in the following recommendations, make sure you understand its options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path
+
+Within a channel each patch release should be directly upgradable to the [HEAD of channel](/docs/glossary/#channel-head). 
+Use skips or skipRange to provide this behaviour. (i.e. if you followed the above `Example 3` that means 
+use skipRange to publish a patch for `3.6.z`under the`stable-v3.6` would mean setting the skipRange to be 
+`>= 3.5.z < 3.6.z`, where `3.5.z` represents the oldest `3.5.z` version for which 
+you can provide direct upgrade support to `3.6.z` latest)
+
+#### Example
+
+Let's imagine that you will publish the Operator bundle `3.6.30` (patch release under the channel `stable-v3.6`) and
+that you have published so far:
+
+- Operator bundles versions from `3.5.0` to `3.5.25` under the `stable-v3.5` channel
+- Operator bundles versions from `3.6.0` to `3.6.29` under the `stable-v3.6` channel
+
+Then, in this case, your newer patch release `3.6.30` would be configured with `skipRange: >=3.5.25 < 3.6.30` in
+order to only supports upgrading to the newest `3.6.z` from a pretty recent `3.5.z`. 
+
+You do not need necessarily to use `3.5.25`. Therefore, you should 
+use the oldest `3.5.z` version that supports upgrades from. Be aware that the ability to configure the path 
+using this example from an Operator version like `3.5.25` _(or `3.5.17`, or something else)_ 
+instead of `3.5.0` is a mechanism to constrain the support matrix when necessary, it's not the default recommendation.
+
+**Note** If you would like to ensure that the users of your Operator still able to install the Operator bundle version
```

Comment:
Wording; "that the users of your Operator still able" -> "that the users of your Operator are still able"